### ISSUE TITLE
Hyperparameter values for ControlNet with semantic segmentation

### ISSUE THREAD

**Issue by Dan-Leh** [lllyasviel/ControlNet#118]
Hi,

I wanted to ask what learning rate and batch size were used to train ControlNet with semantic segmentation for both the ADE20k and COCO datasets? 

Thanks in advance!

**Comment by Dan-Leh** [comment]
@lllyasviel could you also add how many training iterations you used, in addition to learning rate and batch size as asked above?

**Comment by omaralvarez** [comment]
I'm also interested in this.

**Comment by Sierkinhane** [comment]
Is the model trained using MS-COCO available? It seems that `[control_sd15_seg.pth](https://huggingface.co/lllyasviel/ControlNet/blob/main/models/control_sd15_seg.pth)` is the model trained using ADE20K.

**Comment by hxy-123-coder** [comment]
@Sierkinhane @Dan-Leh Hello！Did you train ControlNet with ADE20K ? How the dataset should be organized if using ADE20k?
